### PR TITLE
CA: use expander=prefer-spot

### PR DESCRIPTION
**Pull request by aermakov-zalando**
Let's try running it like this and see if it performs good enough.

### CODE REVIEW

File: cluster/manifests/kube-cluster-autoscaler/daemonset.yaml [hunk 1]
```diff
@@ -41,6 +41,7 @@ spec:
           - --skip-nodes-with-system-pods=false
           - --skip-nodes-with-local-storage=false
           - --scale-up-cloud-provider-template=true
+          - --expander=prefer-spot
```
Comment (mikkeloscar) [inline cluster/manifests/kube-cluster-autoscaler/daemonset.yaml:45]: This flag works in both versions? (You only updated one of them).

**Comment by aermakov-zalando** [comment]
👍

File: cluster/manifests/kube-cluster-autoscaler/daemonset.yaml (hunk 1 above)
Comment (aermakov-zalando) [reply cluster/manifests/kube-cluster-autoscaler/daemonset.yaml:45]: Oops, you're right.

**Comment by aermakov-zalando** [comment]
👍

**Comment by szuecs** [comment]
:+1:

**Comment by linki** [comment]
Related to that: https://github.com/pusher/k8s-spot-rescheduler#deploy-to-kubernetes proposes to set a `PreferNoSchedule` taint on all non-spot workers (https://github.com/pusher/k8s-spot-rescheduler#motivation) so that by default spot instances are filled up first by the scheduler.

@aermakov-zalando @mikkeloscar WDYT?

**Pull request merged**